### PR TITLE
Lesson 1 and Lesson 2 assignments

### PR DESCRIPTION
**Pull request by Swt16**
Lesson 1 and Lesson 2 assignments

### CODE REVIEW

File: students/ShinTran/lesson02/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Lesson 2 Assignment
+'''
+
+# Prints all the numbers from 1 through 100 inclusive
+# For multiples of 3's, it would print Fizz instead
+# For multiples of 5's, it would print Buzz instead
+# For multiples of 3's and 5's, it would print FizzBuzz instead
+def printFizzBuzz():
+	for i in range(1,101):
+		if (i % 3 == 0 and i % 5 == 0):
```
Comment (robalford) [inline students/ShinTran/lesson02/FizzBuzz.py]: The parens are not necessary here

File: students/ShinTran/lesson02/FizzBuzz.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Shin Tran
+Python 210
+Lesson 2 Assignment
+'''
+
+# Prints all the numbers from 1 through 100 inclusive
+# For multiples of 3's, it would print Fizz instead
+# For multiples of 5's, it would print Buzz instead
+# For multiples of 3's and 5's, it would print FizzBuzz instead
+def printFizzBuzz():
```
Comment (robalford) [inline students/ShinTran/lesson02/FizzBuzz.py]: The convention in Python is to use `snake_case` for function and variable names

File: students/ShinTran/lesson02/Series.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	y = 1
+	if n >= 1:
+		print(x)
+	if n >= 2:
+		print(y)
+	for j in range(3,n+1):
+		z = x + y
+		x = y
+		y = z
+		print(z)
```
Comment (robalford) [inline students/ShinTran/lesson02/Series.py]: This is missing the third `sum_series` function for the assignment

**Comment by robalford** [comment]
@Swt16 we still are not seeing your assignment submissions on the course website. You need to submit there as well as on GitHub so that we can grade your assignments. Please let us know if you have any questions about the process. Thanks!

**Comment by natasha-aleksandrova** [comment]
@Swt16 I would check your github account settings (there are some controls for notifications settings)

**Pull request merged**